docbrown/db/benches/base.rs

```diff
@@ -1,19 +1,25 @@
-use crate::common::bootstrap_graph;
+use crate::common::{bootstrap_graph, run_large_ingestion_benchmarks};
 use common::{run_analysis_benchmarks, run_ingestion_benchmarks};
 use criterion::{criterion_group, criterion_main, Criterion, Throughput};
 use docbrown_db::graph_loader::lotr_graph::{self, lotr_graph};
 
 mod common;
 
 pub fn base(c: &mut Criterion) {
-    let mut ingestion_group = c.benchmark_group("ingestion");
-    ingestion_group.throughput(Throughput::Elements(1));
-    run_ingestion_benchmarks(&mut ingestion_group, || bootstrap_graph(4, 10_000), None);
-    ingestion_group.finish();
-
-    let mut analysis_group = c.benchmark_group("analysis");
-    run_analysis_benchmarks(&mut analysis_group, || lotr_graph(4), None);
-    analysis_group.finish();
+    // let mut ingestion_group = c.benchmark_group("ingestion");
+    // ingestion_group.throughput(Throughput::Elements(1));
+    // run_ingestion_benchmarks(&mut ingestion_group, || bootstrap_graph(4, 10_000), None);
+    // ingestion_group.finish();
+    //
+    // let mut analysis_group = c.benchmark_group("analysis");
+    // run_analysis_benchmarks(&mut analysis_group, || lotr_graph(4), None);
+    // analysis_group.finish();
+    let mut large_group = c.benchmark_group("large");
+    large_group.warm_up_time(std::time::Duration::from_secs(1));
+    large_group.sample_size(10);
```

Comment:
Here you can set as well `measurement_time`, otherwise it's going to run for 5 seconds even if you only asked for 10 samples. We can set that to maybe 3 seconds. Besides that, you can set the `throughput`to 10k elements the same way as in line 10, so we have more useful metrics in the report.